### ISSUE TITLE
False positives with 953100

### ISSUE THREAD

**Issue by EsadCetiner** [coreruleset/coreruleset#3304]
### Description
Roundcube has a function that allows users to import emails into their mail account. 953100 is matching the text ``Maximum allowed file size is 10 MB`` and is loaded everytime the mailbox is opened.
I've tuned away this false positive but this seems like something that can be very common.

### Logs

```
Maximum allowed file size is 10 MB
```
<b>Matched Data:</b> file size is

### Your Environment

* CRS version (e.g., v3.3.4): CRS 4 Dev Latest
* Paranoia level setting (e.g. PL1) : 1
* ModSecurity version (e.g., 2.9.6): libmodsecurity 3.0.10
* Web Server and version or cloud provider / CDN (e.g., Apache httpd 2.4.54): Nginx
* Operating System and version: Ubuntu

### Confirmation

[x] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by franbuehler** [comment]
Hi!

Thanks a lot for your report.

I can't reproduce your issue. When I test your string `Maximum allowed file size is 10 MB` against the regex of rule 953110 I don't see a match:
https://regex101.com/r/PtSVnW/1

This would perfectly fit into a Roundcube exclusion plugin. As long as we don't have such an exclusion package, I don't think we'll adapt the current rule 953110.

**Comment by dune73** [comment]
In fact @EsadCetiner is contributing several Roundcube exclusion packages.

https://github.com/coreruleset/plugin-registry

If we can reproduce I do not rule out patching the core rule set rule itself, since this sounds so generic. Can you help us here Esad?

**Comment by EsadCetiner** [comment]
@dune73 @franbuehler Sorry, I meant the rule 953100. Please see https://github.com/coreruleset/coreruleset/blob/1fe255d06a282927fce55d6907a497b35a0e7a96/rules/php-errors.data#L1785

**Comment by franbuehler** [comment]
Ah! This exclusion package is not yet in our coreruleset repository, therefore I didn't find it.

But, yes! Then this help would be very welcome!

**Comment by EsadCetiner** [comment]
I just want to clarify the FP is already fixed in the plugin, but ideally it should be fixed within CRS.

**Comment by dune73** [comment]
Thank you for this essential context.

**Comment by theseion** [comment]
@EsadCetiner, we discussed your finding in the team and we agree that it should be fixed in CRS. We think that the phrase should be moved to the file `php-errors-pl2.data`, that already contains some of the error strings that are prone to FPs at paranoia level 1.

Would you be willing to create a PR?

**Comment by dune73** [comment]
@M4tteoP has been very eager to get this done. He has provided a PR in the meantime. See #3314.

**Comment by EsadCetiner** [comment]
Looks like he beat me to it 😄 
If there's nothing wrong with the PR then I don't see why it shouldn't be merged.

**Comment by M4tteoP** [comment]
Adding just some more tests right now :)

**Comment by dune73** [comment]
He's a fast one.
